modules/gapi/test/common/gapi_imgproc_tests_inl.hpp

```diff
@@ -456,6 +456,29 @@ TEST_P(YUV2RGBTest, AccuracyTest)
     }
 }
 
+TEST_P(YUV2GrayTest, AccuracyTest)
+{
+    // G-API code //////////////////////////////////////////////////////////////
+    cv::GMat in;
+    auto out = cv::gapi::YUV2Gray(in);
+
+    cv::Mat in_mat(cv::Size(sz.width, static_cast<int>(sz.height * 1.5)), CV_8UC1);
```

Comment:
I believe this function is misaligned with https://docs.opencv.org/4.2.0/dc/d38/group__gapi__colorconvert.html#gad322eb34ae0d3ca84e4666fd5351bec9 which takes an interleaved YUV444 image.

If you've had I420 in mind, then the kernel needs to be renamed appropriately and have 3 inputs for every plane.

Reply:
Done